static/pages/home/home.html

```diff
@@ -340,28 +334,24 @@ <h3 class="md-display-1">
            layout-fill>
     <md-card-content class="home-page-display-size
                             oppia-foundation-no-padding"
-                     layout="column"
-                     layout-align="center center">
+                      layout="column" 
+                      layout-align="center center">
       <md-content class="oppia-foundation-black-text
                          oppia-foundation-white-background
-                         home-page-description-text-box"
+                         home-page-donation-text-box" 
```

Comment:
I think this is a left-over from the git conflict. Can you check if this css class still exists?